apps/store/modules/asset-api.js

```diff
@@ -540,4 +540,24 @@ var responseProcessor = require('utils').response;
         paging.url = url;
         return paging;
     };
+    
+    api.formatSearchQuery = function (queryString) {
+        var searchQuery = "";
+        var qjson = parse('{' + queryString + '}');
+
+        var searchKeys = Object.keys(qjson);
+        if ((searchKeys.length === 1) && (searchKeys.indexOf("name") >= 0)) {
+            searchQuery += qjson[searchKeys.pop()];
+        }
+        else {
```

Comment:
Thanks @splinter , This method *formatSearchQuery* has been moved to *request.js* from [this commit](https://github.com/wso2/carbon-store/commit/8e2edd3139a54ee3b1571eacabbffcc7ccad25d3#diff-26fa7e5ae96217c017cd62b4d21baee7), but this formatting issue was still there. I have fixed it in [this commit](https://github.com/wso2/carbon-store/pull/844/commits/1cfd4a532c0b6fcbb158ab5da98f004afac2612c)